lib/app/features/feed/stories/views/components/story_capture/camera/idle_camera_preview.dart

```diff
@@ -57,10 +60,21 @@ class IdleCameraPreview extends ConsumerWidget {
                 final mediaFiles =
                     await MediaPickerRoute(maxSelection: 1).push<List<MediaFile>>(context);
                 if (mediaFiles != null && mediaFiles.isNotEmpty && context.mounted) {
-                  await StoryPreviewRoute(
-                    path: mediaFiles.first.path,
-                    mimeType: mediaFiles.first.mimeType,
-                  ).push<void>(context);
+                  var filePath =
+                      await ref.read(assetFilePathProvider(mediaFiles.first.path).future);
+
+                  if (filePath == null) return;
```

Comment:
Maybe show some error to the user?

Reply:
for now there is no such a product request. Also keep in mind, PO ll check this package it is not considired we ll use 100%.